website/allowed-subcategories.txt

```diff
@@ -110,6 +110,7 @@ SimpleDB
 Step Function (SFN)
 Storage Gateway
 Synthetics
+TimestreamWrite
```

Comment:
Human facing category can have the space in the name. 😄 

```suggestion
Timestream Write
```

Reply:
Will do!